app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,84 @@
+class SessionsController < ApplicationController
+    register Sinatra::ActiveRecordExtension
+    set :views, Proc.new { File.join(root, "../views/") }
+
+    configure do
+      enable :sessions
+      set :session_secret, "secret"
+    end
```

Comment:
We don't need to set up configurations like this outside of ApplicationController